graal/backends/core/analyzers/lizard.py

```diff
@@ -94,3 +88,60 @@ def analyze(self, **kwargs):
 
         result['funs'] = funs_data
         return result
+
+    def __analyze_repository(self, repository_path, details):
+        """Add code complexity information for a given repository
+        using Lizard and CLOC.
+
+        Current information includes cyclomatic complexity (ccn),
+        lines of code, number of functions, tokens, blanks and comments.
+
+        :param repository_path: repository path
+        :param details: if True, it returns fine-grained results
+
+        :returns  result: list of the results of the analysis
+        """
+        analysis_result = []
+
+        repository_analysis = lizard.analyze(
+            paths=[repository_path],
+            threads=1,
+            exts=lizard.get_extensions([]),
+        )
+        cloc = Cloc()
+
+        for analysis in repository_analysis:
+            cloc_analysis = cloc.analyze(file_path=analysis.filename)
+            result = {
+                'loc': analysis.nloc,
+                'ccn': analysis.CCN,
+                'tokens': analysis.token_count,
+                'num_funs': len(analysis.function_list),
+                'file_path': analysis.filename,
```

Comment:
The local repository path should be stripped out, for instance with:
`'file_path': analysis.filename.replace(repository_path, '')`

WDYT?

Reply:
Yes. It should be! 
I'll update the PR with the change